lib/runner.js

```diff
@@ -22,11 +22,16 @@ function runner(options, next) {
 
     /*jslint unparam:true*/
     exec(args.join(' ').trim(), function (err, stdout, stderr) {
-
       return parseOutput(stderr, next);
+    }).on('exit', function (exitCode) {
+      process.exitCode = exitCode;
     });
     /*jslint unparam:false*/
   });
 }
 
+process.on('exit', function () {
+  process.exit(process.exitCode);
```

Comment:
No need to do this. Setting `process.exitCode` above is enough.

https://nodejs.org/docs/latest/api/process.html#process_process_exitcode

> __process.exitCode__
> A number which will be the process exit code, when the process either exits gracefully